async.go

```diff
@@ -56,10 +56,7 @@ func processAsyncResults() {
 			ro, err := cache.Profiles().ResourceOperation(device.Profile.Name, cv.DeviceResourceName, common.GetCmdMethod)
 			if err != nil {
 				common.LoggingClient.Error(fmt.Sprintf("processAsyncResults - getting resource operation failed: %s", err.Error()))
```

Comment:
So the Error will be logged, but the result will be successful processed? Should this be a Debug message?

Reply:
Thanks for the reminder, I'll fix this.